simpl/webhook/tasks.py

```diff
@@ -0,0 +1,50 @@
+import requests
+
+from django.apps import apps
+from django.utils import timezone
+from celery import shared_task
+
+
+@shared_task
+def send_webhook(event, url, payload, connected, erroring, subscriber_id):
+    """
+    Send a HTTP POST (webhook) with the payload to the subscribed URL.
+
+    We are also passed whether or not this subscriber has been connected
+    before, whether or not they have been erroring lately and the subscriber_id.
+
+    We use these to determine if we need to retrieve the subscriber object
+    from the database and update the status.  So for example, if they're
+    already showing they are connected we don't need to update that except for
+    the first successful connection.
+
+    In the case of errors, we want to update the database when we see a
+    successful webhook happen.  To avoid blocking non-erroring tasks we assume
+    the dispatch call for this task sets a small ETA countdown on the Celery
+    task to help ensure erroring tasks do not clog the Celery broker's queue.
+    """
```

Comment:
Do we also want to add retries for just network issues?

Reply:
I think we should maybe consider that in the future, I'm a little worried about "out of order" update scenarios and how that might impact the modelservice's "view" of what the up to date data is.